source_code/main_mcu/src/GUI/gui_prompts.c

```diff
@@ -576,6 +576,37 @@ void gui_prompts_display_3line_information_on_screen_and_wait(confirmationText_t
     }
 }
 
+/*! \fn     gui_prompts_get_char_to_display(uint8_t const * current_pin, uint8_t index)
+*   \brief  Get the next PIN value to display or '*' if PIN should be hidden
```

Comment:
add: "called whenever a non selected PIN digit needs to be displayed"

Reply:
Done!